### PR TITLE
CODE RUB: Separate Logic Test For Create Operation - Teachers

### PR DESCRIPTION
**Pull request by tharanisantosh**

### CODE REVIEW

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]
Got conflicts. Address the title and link the issue too.

**Comment by ShriHumrudha** [comment]
Beyond the 3 day window.

**Comment by ShriHumrudha** [comment]
@tharanisantosh reopening as per your request

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Logic.Create.cs [hunk 1]
```diff
@@ -8,6 +8,11 @@
 using FluentAssertions;
 using Moq;
 using OtripleS.Web.Api.Models.Teachers;
+using System;
```
Comment (ShriHumrudha) [inline OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Logic.Create.cs:11]: Run code cleanup

**Review by ShriHumrudha — CHANGES_REQUESTED** [review]

**Review by ShriHumrudha — APPROVED** [review]
@tharanisantosh use appropriate PR title

**Comment by tharanisantosh** [comment]
> @tharanisantosh use appropriate PR title

Ok Humrudha. Noted.

File: OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Logic.Create.cs (hunk 1 above)
Comment (tharanisantosh) [reply OtripleS.Web.Api.Tests.Unit/Services/Foundations/Teachers/TeacherServiceTests.Logic.Create.cs]: Code cleanup completed.

**Pull request merged**